### ISSUE TITLE
Transpiled files are missing in released gem

### ISSUE THREAD

**Issue by haines** [palkan/action_policy#133]
### Tell us about your environment

**Ruby Version:** 2.6.6

**Framework Version (Rails, whatever):** Rails 6.0.3.3

**Action Policy Version:** 0.5.0

**Reproduction Script:** 
```ruby
# frozen_string_literal: true

require 'bundler/inline'

gemfile do
  source 'https://rubygems.org'
  gem 'action_policy', '0.5.0'
end
```

```console
$ ruby test.rb
Traspiled files are missing in: /Users/andrew/.gem/ruby/2.6.6/gems/action_policy-0.5.0/lib/.rbnext. 
Make sure you have gem 'ruby-next' in your Gemfile to auto-transpile the required files from source on load. Otherwise the code from /Users/andrew/.gem/ruby/2.6.6/gems/action_policy-0.5.0 may not work correctly.
```

### What did you do?
Installed action_policy 0.5.0

### What did you expect to happen?
It doesn't require me to install ruby-next myself because the transpiled files are shipped in the gem, as [described here](https://github.com/ruby-next/ruby-next#integrating-into-a-gem-development).

### What actually happened?
The transpiled files aren't included in the gem, so it can't be loaded.

**Comment by palkan** [comment]
My bad 😞 Thanks for catching this!
Released 0.5.1 with the transpiled files included.

**Comment by haines** [comment]
No worries! Thanks for fixing so quickly, we're looking forward to taking v0.5 for a spin - ability to use rule aliases more within policies is a great feature 😊
